src/Symfony/Component/Messenger/composer.json

```diff
@@ -35,7 +36,8 @@
         "symfony/var-dumper": "~3.4|~4.0"
     },
     "conflict": {
-        "symfony/event-dispatcher": "<4.3"
+        "symfony/event-dispatcher": "<4.3",
+        "symfony/debug": "<4.1"
```

Comment:
Because, IF symfony/debug is available, we need the `FlattenException::createFromThrowable()` method.